dockernet/scripts/community-pool-staking/reinvest.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -eu 
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source ${SCRIPT_DIR}/../../config.sh
+
+echo ">>> Sending usdc tokens to withdrawal ICA to simulate rewards..." 
+$NOBLE_MAIN_CMD tx ibc-transfer transfer transfer channel-0 $(GET_ICA_ADDR DYDX withdrawal) 1000000${USDC_DENOM} \
```

Comment:
You know it will be noble's channel-0 for the transfer to dydx because of the order you made the connecting relayers?

Reply:
yep exactly! we could also try and determine it dynamically but it felt like doing a bit too much for a simple script like this

Reply:
yeah I hear you, but the ordering that channels are created seems like one of the easiest things to break by accident.  And some of the code might still work, it would just be interacting with a different unintended chain so it would be harder to catch why certain integration tests suddenly start failing